homeassistant/components/withings/__init__.py

```diff
@@ -171,6 +172,11 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     return True
 
 
+async def update_listener(hass: HomeAssistant, entry: ConfigEntry) -> None:
+    """Handle options update."""
+    await hass.config_entries.async_reload(entry.entry_id)
```

Comment:
I think you are going to get a duplicate reload on reauth now since these lines change the config entry data and there is also a reload after it in the config flow

```
            self.hass.config_entries.async_update_entry(self.reauth_entry, data=data)
            await self.hass.config_entries.async_reload(self.reauth_entry.entry_id)
```

Reply:
Should I remove it from reauth?